packages/lib/models/Folder.test.ts

```diff
@@ -353,4 +353,17 @@ describe('models/Folder', () => {
 		await expectThrow(async () => Folder.delete(folder2.id, { toTrash: true, toTrashParentId: folder2.id }));
 	});
 
+	it('should tell if atleast one folder other than trash and deleted exists', async () => {
```

Comment:
Should be "at least" (not sure why "atleast" was in our dictionary but I removed it now)